samcli/commands/init/interactive_init_flow.py

```diff
@@ -40,29 +45,90 @@ def do_interactive(
     It will ask customers a few questions to init a template.
     """
     if app_template:
-        location_opt_choice = "1"
+        location_opt_choice = "2"
     else:
         click.echo("Which template source would you like to use?")
-        click.echo("\t1 - AWS Quick Start Templates\n\t2 - Custom Template Location")
-        location_opt_choice = click.prompt("Choice", type=click.Choice(["1", "2"]), show_choices=False)
-    if location_opt_choice == "2":
+        click.echo("\t1 - Hello World application\n\t2 - AWS Quick Start Templates\n\t3 - Custom Template Location")
+        location_opt_choice = click.prompt("Choice", type=click.Choice(["1", "2", "3"]), show_choices=False)
+
+        get_start_options(
+            location,
+            pt_explicit,
+            package_type,
+            runtime,
+            base_image,
+            dependency_manager,
+            output_dir,
+            name,
+            app_template,
+            no_input,
+            location_opt_choice,
+        )
+
+
+def get_start_options(
+    location,
+    pt_explicit,
+    package_type,
+    runtime,
+    base_image,
+    dependency_manager,
+    output_dir,
+    name,
+    app_template,
+    no_input,
+    location_opt_choice,
+):
+    if location_opt_choice == "1":
+        _generate_simple_application(
+            location, pt_explicit, runtime, base_image, dependency_manager, output_dir, name, app_template, package_type
+        )
+
+    elif location_opt_choice == "2":
+        _generate_from_use_case(
+            location, pt_explicit, package_type, runtime, base_image, dependency_manager, output_dir, name, app_template
+        )
+
+    else:
         _generate_from_location(
             location, package_type, runtime, dependency_manager, output_dir, name, app_template, no_input
         )
-    else:
-        if not pt_explicit:
-            click.echo("What package type would you like to use?")
-            click.echo("\t1 - Zip (artifact is a zip uploaded to S3)\t")
-            click.echo("\t2 - Image (artifact is an image uploaded to an ECR image repository)")
-            package_opt_choice = click.prompt("Package type", type=click.Choice(["1", "2"]), show_choices=False)
-            if package_opt_choice == "1":
-                package_type = ZIP
-            else:
-                package_type = IMAGE
-
-        _generate_from_app_template(
-            location, package_type, runtime, base_image, dependency_manager, output_dir, name, app_template
+
+
+# pylint: disable=too-many-statements
+def _generate_simple_application(
+    location, pt_explicit, runtime, base_image, dependency_manager, output_dir, name, app_template, package_type=ZIP
+):
+    if pt_explicit and package_type == IMAGE:
+        raise InvalidInitOptionException("Hello World Application only supports ZIP package type")
```

Comment:
should we add something about, what to do for IMAGE as well?

Reply:
we current do not support Images for the hello application. Customers can use image in Quick start template. During the last UX review this was agreed on this but if we need to pass a message about images we might need to reachout to Molly again.

Reply:
I think what Sriram meant was that we may need a message to point customers in the right direction to use an image instead of just telling them "Not supported". We will see this with UX.

Reply:
Do you mean information telling customer about Images?

Reply:
@jonife Yes, as @mgrandis pointed out, that is where I was going at.

Reply:
Molly is currently away, she should be back on monday.